cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -135,4 +135,10 @@
    * Option for requirements of various plugins present in the program
    */
   public static final String PLUGIN_REQUIREMENTS = "pluginRequirements";
+
+  /**
+   * Runtime arguments for program impersonation
+   */
+  public static final String RUNTIME_ARG_KEYTAB = "runtime.keytab.path";
```

Comment:
Since this is system runtime arguments, please define them in `SystemArguments` and prefix it with `system.`, e.g. `system.runtime.keytab.path`.

Reply:
Changes done. Please check.